### PR TITLE
Update ImpeREAL Empire Unique Cities - Sutch

### PR DESCRIPTION
**Pull request by charabaruk**
* Relocate to appropriate master list section
  - Locations - New Structures & Landmarks

* Add group
  - Cities

# Sutch Coblized #
* Add group
  - Late Patches

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -18506,6 +18506,17 @@ plugins:
       - crc: 0xEDC9F9B4
         util: 'TES4Edit v4.0.3h (Hotfix 1)'
 
+  - name: 'ImpeREAL Empire Unique Cities - Sutch.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/37867/'
+        name: 'ImpeREAL Empire Unique Cities - Sutch'
+    group: *citiesGroup
```
Comment (sibir-ine) [inline masterlist.yaml:18513]:
```suggestion
```

The mod adds a new city, so it seems very unlikely that it'd conflict with many other mods & require priority in most situations. To the extent that it does edit vanilla content, it's a few references, landscapes, & path grids, presumably within range of the city. If these do conflict with other mods, patches or specific load after rules would be more appropriate than a group rule. The default group seems appropriate for this mod.

File: masterlist.yaml [hunk 2]
```diff
@@ -34174,20 +34185,11 @@ plugins:
...
-      - <<: *quickClean
-        crc: 0xEB59AB31
-        util: '[TES4Edit v4.0.4](https://www.nexusmods.com/oblivion/mods/11536)'
-        itm: 2
-
   - name: 'Sutch Coblized.esp'
     url:
       - link: 'https://www.nexusmods.com/oblivion/mods/52617/'
         name: 'Sutch Coblized'
+    group: *latePatchesGroup
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
```

The **Late Patches** group predates me, but it seems to primarily be used for filter patches & a few other patches. This mod only adds references to a city added by another mod, so the chances of conflicts are already extremely low. They'd also presumably be conflicts like collisions with other mod-placed references which cannot be resolved with load order, so the default group seems appropriate here. Also, this should've been added in a separate PR as these are separate mods from separate pages (despite one being an add-on for another), although that won't be necessary anymore as the change is being rejected.

**Review by sibir-ine — CHANGES_REQUESTED** [review]

File: masterlist.yaml (hunk 2 above)
Comment (charabaruk) [reply masterlist.yaml]: If Sutch is removed from the cities group, than this change isn't even required at all. It was added to break a cycle caused by putting ImpeREAL Empire Unique Cities - Sutch into the cities group.

**Comment by charabaruk** [comment]
Rolled back branch to f650e93

**Review by sibir-ine — APPROVED** [review]

**Pull request merged**